### PR TITLE
Add a built-in thermometer to RM13 armor

### PR DESCRIPTION
**Pull request by BrettDong**
Idea suggested by a player. There should be a built-in thermometer in RM13 armor, telling current temperature to the player continuously.

### CODE REVIEW

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -593,7 +593,7 @@
         "name": "RM13 combat armor (on)",
         "name_plural": "RM13 combat armors (on)",
         "description": "Internally powered by a maximum of ten plutonium fuel cells, this full-body suit of sleek black military armor represents the pinnacle of Rivtech's non-rigid powered armor technology.  It is turned on, and continually draining power.  Use it to turn it off.",
-        "flags" : ["VARSIZE", "STURDY", "WATERPROOF", "RAINPROOF", "WATCH", "ALARMCLOCK", "SWIM_GOGGLES", "SUN_GLASSES", "RAD_PROOF", "ELECTRIC_IMMUNE", "TRADER_AVOID"],
+        "flags" : ["VARSIZE", "STURDY", "WATERPROOF", "RAINPROOF", "WATCH", "ALARMCLOCK", "SWIM_GOGGLES", "SUN_GLASSES", "RAD_PROOF", "ELECTRIC_IMMUNE", "THERMOMETOR", "TRADER_AVOID"],
```
Comment (Rivet-the-Zombie) [inline data/json/items/tool_armor.json]: Flags don't work right if they're misspelled. I'm guessing this was never actually tested?

**Comment by BorkBorkGoesTheCode** [comment]
@BrettDong the flag is misspelled: it should be `THERMOMETER`

**Comment by BrettDong** [comment]
So sorry for the stupid mistake. Did test locally, but the commit was directly written on the web, neglected the spelling.

**Comment by BorkBorkGoesTheCode** [comment]
@BrettDong You haven't damaged anything, so you're okay.

**Pull request merged**